docroot/sites/iisc.uiowa.edu/modules/iisc_core/templates/views-view-unformatted--partners-recent.html.twig

```diff
@@ -0,0 +1,69 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ *
+ * @ingroup themeable
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+{% for key,row in rows %}
+  {% set row_classes = [
+    default_row_class ? 'views-row',
+  ] %}
```

Comment:
I think this can be moved outside of the for loop since the `default_row_class` appears to be a top-level variable rather than something that changes per row.